x-pack/legacy/plugins/apm/public/hacks/toggle_app_link_in_nav.ts

```diff
@@ -6,9 +6,9 @@
 
 import { npStart } from 'ui/new_platform';
```

Comment:
Is this still allowed in NP?

Reply:
the hack pattern will go away eventually, but the `npStart` import is valid until we get to that point. This is how legacy plugins import the new platform core at the top-level.